conan/tools/apple/apple.py

```diff
@@ -30,6 +33,30 @@ def to_apple_arch(conanfile, default=None):
     return _to_apple_arch(arch_, default)
 
 
+def _to_apple_archs(conanfile, default=None):
```

Comment:
This ";" separated list is specific to CMakeToolchain, or is it valid for other build systems? If it is only CMakeToolchain, maybe move it there?

Reply:
Yes, better move it to the CMakeToolchain as this is CMake syntax 👍